<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>260</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>501</width>
      <height>261</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_sections">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Sections</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_sections">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_Plus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_Minus">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>501</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_add">
       <property name="title">
        <string>Additing</string>
       </property>
       <widget class="QLabel" name="label_type">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>50</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_key_add">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>50</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Key</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_value">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>50</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Value</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_type">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>30</y>
          <width>180</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_key">
        <property name="geometry">
         <rect>
          <x>62</x>
          <y>60</y>
          <width>180</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_value">
        <property name="geometry">
         <rect>
          <x>62</x>
          <y>90</y>
          <width>180</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="button_add">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_del">
       <property name="title">
        <string>Deleting</string>
       </property>
       <widget class="QComboBox" name="comboBox_key">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>30</y>
          <width>180</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_key_del">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>50</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Key</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button_delete">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_file"/>
    <addaction name="actionOpen_directory"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_file"/>
   <addaction name="actionOpen_directory"/>
   <addaction name="actionSave"/>
  </widget>
  <action name="actionOpen_file">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/img/open_file.png</normaloff>:/images/img/open_file.png</iconset>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionOpen_directory">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/img/open_directory.png</normaloff>:/images/img/open_directory.png</iconset>
   </property>
   <property name="text">
    <string>Open directory</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/img/save.png</normaloff>:/images/img/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
